tests/context-menu.spec.ts

```diff
@@ -1,4 +1,262 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+import {
+  makeTableCommand,
+  pasteInMonaco,
+  TableTypes,
+  waitForLoadingDone,
+  generateVarName,
+} from './utils';
+
+test.describe.configure({ mode: 'serial' });
+
+async function openAdvancedFilters(page: Page) {
+  await page
+    .locator('.iris-grid .grid-wrapper')
+    .click({ button: 'right', position: { x: 20, y: 20 } });
+  await page.getByRole('button', { name: 'Advanced Filters' }).click();
+}
+
+async function moveMouseAwayFromTable(page: Page) {
+  await page.mouse.move(0, 0);
+  await page.mouse.click(0, 0);
+}
```

Comment:
Similarly with this method - you're using it to move the mouse away from the table, but the actual function is just trying to reset the mouse position and focus. I'd probably do something like:
```
async function resetMousePosition(page: Page, resetFocus = true) {
  await page.mouse.move(0, 0);
  if (resetFocus) {
    await page.mouse.click(0, 0);
  }
}
```